tests/js/testTimestamps.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-env jasmine */
```

Comment:
I'm afraid not :( ESLint seems to only detect it if it is in a C style comment (tested again now).